app/grandchallenge/challenges/emails.py

```diff
@@ -42,3 +44,133 @@ def send_external_challenge_created_email(challenge):
         subject=f"[{site.domain.lower()}] New External Challenge",
         message=message,
     )
+
+
+def send_challenge_requested_email_to_reviewers(challengerequest):
+    site = Site.objects.get_current()
+    update_url = reverse(
+        "challenges:request-list",
+    )
+    message = (
+        f"Dear reviewers,\n\n"
+        f"User {challengerequest.creator} has just requested the challenge "
+        f"{challengerequest.title}. To review the challenge, go here:"
+        f"{update_url}\n\n"
+        f"Regards,\n"
+        f"{site.name}\n\n"
+        f"This is an automated service email from {site.domain}."
+    )
+    reviewers = (
+        Group.objects.filter(name=settings.CHALLENGE_REVIEWERS_GROUP_NAME)
+        .get()
+        .user_set.all()
+    )
+
+    send_mail(
+        subject=f"[{site.domain.lower()}] New Challenge Requested",
+        message=message,
+        from_email=settings.DEFAULT_FROM_EMAIL,
+        recipient_list=[user.email for user in reviewers],
+    )
+
+
+def send_challenge_requested_email_to_requester(challengerequest):
+    site = Site.objects.get_current()
+    addition = ""
+    if (
+        challengerequest.challenge_type
+        == challengerequest.ChallengeTypeChoices.T2
+    ):
+        budget = ""
+        for key, value in challengerequest.budget.items():
+            budget += f"{key}: {value}\n"
+        addition += (
+            f"For your type 2 challenge, we have calculated the following budget estimate "
+            f"based on the information you provided:\n"
+            f"{budget}\n\n"
+        )
+
+    message = (
+        f"Dear {challengerequest.creator.username},\n\n"
+        f"Your challenge request has been sent to the reviewers. You will "
+        f"receive an email informing you of our decision within the next 4 weeks. "
+        f"The reviewers might contact you for additional information during "
+        f"that time.\n"
+        f"{addition}"
+        f"Regards,\n"
+        f"{site.name} team\n\n"
+        f"This is an automated service email from {site.domain}."
+    )
+
+    send_mail(
+        subject=f"[{site.domain.lower()}] Challenge Request Submitted Successfully",
+        message=message,
+        from_email=settings.DEFAULT_FROM_EMAIL,
+        recipient_list=[challengerequest.creator.email],
+    )
+
+
+def send_challenge_status_update_email(challengerequest, challenge=None):
+    site = Site.objects.get_current()
+    message = ""
+    if (
+        challengerequest.status
+        == challengerequest.ChallengeRequestStatusChoices.ACCEPTED
+    ):
+        message = (
+            f"Dear {challengerequest.creator.username},\n\n"
```

Comment:
These emails are currently sent plain text, but it is good practise to always use `format_html` to automatically escape any user input when building templates. You can see these reflective things in a lot of the spam email, where spammers create accounts with usernames like `<a href="my-phising-site">CLICK ME</a>` etc etc.